xv83.md

```diff
@@ -35,16 +27,47 @@ xv83/
 │   ├── kd7073/
 │   ├── mxt599/
 │   └── tm4888/
+
+xv84/
+├── admin/
+├── shared/
+|   ├── publications/
+│   ├── authoritative/
+│   │   ├── CAFE/
+│   │   │   ├── README
+│   │   │   ├── CAFE60v1/
+│   │   │   ├── CAFE-f5/
+│   │   │   └── CAFE-f6/
+│   ├── reference_datasets/
+|   │   ├── replicas/
+|   │   │   ├── CCI_fire/
+|   │   │   │   ├── README
+|   │   │   │   └── *.nc
+|   │   ├── post-processed/
+|   │   │   ├── AWAP/
+|   │   │   │   ├── code/
+|   │   │   │   └── data/
+|   │   │   ├── HadISST/
+|   │   │   │   ├── code/
+|   │   │   │   └── data/
+|   │   │   └── JRA55/
+|   │   │   │   ├── code/
+|   │   │   │   └── data/
+
+xv85/
+xv86/
+etc
 ```
+What is described above is essentially a pipeline that goes from 'working space' (`xv83`) to 'shared space' (`xv84`, i.e. soft published data) to 'managed collection/s' (`xv85`, `xv86` onwards, i.e. hard published).
```

Comment:
@chloemackallah Do we need to describe what exactly a managed collection is?

Reply:
Yes worth doing.
Can also link to: https://opus.nci.org.au/display/NDP/NCI+Data+Collections+and+Publishing

Reply:
Thanks for the link. 

Just to clarify, authoritative datasets in `/g/data/xv84/authoritative` (i.e. the shared space) wouldn't appear in the NCI data catalogue (or have a DOI) - they'd need to be published as a managed NCI data collection (which we're suggesting would be xv85, xv86 etc but could use different letters and numbers) before that would happen?

Reply:
Thanks, @chloemackallah.

Would a dataset in `/g/data/xv84/publications` appear on the [NCI data catalogue](https://geonetwork.nci.org.au), whereas one in `/g/data/xv84/authoritative` wouldn't? Or would nothing in `xv84` be on the catalogue? Or everything?

Reply:
Those are all really good questions!...I need to revise this now for sure
I think (we need to chat about details though!) nothing in 'xv84' would be in the NCI catalogue, but everything in 'xv85' would be. Typically something needs to be an NCI-managed data collection to be in the NCI catalogue, so I would expect this to be the hard-publication line, which would include a DAP entry. 
Then when something moves from xv84 to xv85, we back it up (CSIRO datastore; although we should have already by this point) and leave the xv85 version as 'authoritative'.

Reply:
Cool. I'm part way through an edit of the document, so I'll commit my changes in a few minutes and then you can take a look and see if I've captured everything.